app/assets/javascripts/discourse/widgets/hamburger-categories.js.es6

```diff
@@ -51,14 +51,15 @@ export default createWidget("hamburger-categories", {
 
   html(attrs) {
     const href = Discourse.getURL("/categories");
+    let title = I18n.t("filters.categories.title");
+    if (attrs.moreCount > 0) {
+      title += I18n.t("categories.more", { count: attrs.moreCount });
```

Comment:
@vinothkannans I must have missed this. Concatenating translations in code is never a good idea. It makes it hard to translate in some languages.